doc/rados/operations/erasure-code.rst

```diff
@@ -212,22 +212,23 @@ mode so that every write and read to the *ecpool* are actually using
 the *hot-storage* and benefit from its flexibility and speed.
 
 More information can be found in the `cache tiering
-<../cache-tiering>`_ documentation.
+<../cache-tiering>`_ documentation.  Note however that cache tiering
+is deprecated and may be removed completely in a future release.
 
 Erasure coded pool recovery
 ---------------------------
-If an erasure coded pool loses some shards, it must recover them from the others.
-This generally involves reading from the remaining shards, reconstructing the data, and
-writing it to the new peer.
-In Octopus, erasure coded pools can recover as long as there are at least *K* shards
+If an erasure coded pool loses some data shards, it must recover them from others.
+This involves reading from the remaining shards, reconstructing the data, and
+writing new shards.
+In Octopus and later releases, erasure coded pools can recover as long as there are at least *K* shards
```

Comment:
What do you think of this?

s/erasure coded pools/erasure-coded pools/

Reply:
I often prefer hyphenates over not.  Given prior debates over _filesystem_ and _backend_ I hadn't gone there with this, but will push this change.